opencti-platform/opencti-front/src/private/components/analyses/reports/ReportCreation.tsx

```diff
@@ -262,6 +266,14 @@ export const ReportCreationForm: FunctionComponent<ReportFormProps> = ({
             containerStyle={fieldSpacingContainerStyle}
             multiple={true}
           />
+          <OpenVocabField
```

Comment:
The UI order is F-E-F-C-B-A
Maybe is better to have A-B-C... ?

If it's the case, I don't know if it's better to change the hardcoded order of this vocab or the back-end sorting

Reply:
Yes I was wondering about the order, and I followed the "logic" of the other vocab, the order was always from worst to better, but I guess in this case with "A B C D E F", it's a bit weird, so maybe it's better to change this order.

Reply:
I changed the order